### PR TITLE
Basic brb feature with a simple text based feedback dialog on chat room

### PR DESCRIPTION
**Pull request by rvenugopal-msft**
## Purpose

* ...

image

image

image

Written to Blob Storage
image

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone https://github.com/Azure-Samples/communication-services-web-chat-hero.git
cd communication-services-web-chat-hero
git checkout [rvenugopal-msft/addTextFeedbackBRB]
npm install
```

* Test the code

```
```

## What to Check
Check that in the chat thread, the Report a bug button is available and the user can report a feedback through a dropdown and textbox.
* ...

### CODE REVIEW

File: Chat/appsettings.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-    "ResourceConnectionString": "REPLACE_WITH_CONNECTION_STRING"
+  "Logging": {
+    "LogLevel": {
+      "Default": "Trace",
+      "System": "Information",
+      "Microsoft": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "ResourceConnectionString": "REPLACE_WITH_CONNECTION_STRING",
```
Comment (PorterNan) [inline Chat/appsettings.json:10]: Let's add a config to enable and disable the button? Could be either here or next PR, up to you

File: Chat/ClientApp/src/components/ChatHeader.tsx [hunk 2]
```diff
@@ -54,7 +56,10 @@ export default (props: ChatHeaderDispatchProps & ChatHeaderProps & ChatHeaderPan
   }, [topic, generateHeaderMessage]);
 
   return (
-    <Stack className={chatHeaderContainerStyle} horizontal={true} horizontalAlign="space-between">
+      <Stack className={chatHeaderContainerStyle} horizontal={true} horizontalAlign="space-between">
+      <Stack.Item grow={1} className={feedbackContainer}>
+        <FeedbackButton />
```
Comment (PorterNan) [inline Chat/ClientApp/src/components/ChatHeader.tsx:61]: Indent looks a little bit off, please fix the indent for the top one 'stack'

**Review by PorterNan — APPROVED** [review]
Awesome, glad that you are making this progress so fast!

File: Chat/appsettings.json (hunk 1 above)
Comment (rvenugopal-msft) [reply Chat/appsettings.json:10]: yes we are planning of adding that in another PR.

File: Chat/ClientApp/src/components/ChatHeader.tsx (hunk 2 above)
Comment (rvenugopal-msft) [reply Chat/ClientApp/src/components/ChatHeader.tsx:61]: will fix that and push the update out.

Comment (rvenugopal-msft) [reply Chat/ClientApp/src/components/ChatHeader.tsx:61]: Fixed the indentation issue.

**Pull request merged**